packages/ui/src/elements/WhereBuilder/Condition/DefaultFilter/index.tsx

```diff
@@ -24,7 +24,7 @@ type Props = {
   onChange: React.Dispatch<React.SetStateAction<string>>
   operator: Operator
   options: Option[]
-  value: string
+  value: unknown
```

Comment:
Is this a necessary evil? or could we type as `string | Date | number | number[]`?